bugzilla/_backendrest.py

```diff
@@ -107,9 +107,17 @@ def bug_create(self, paramdict):
     def bug_fields(self, paramdict):
         return self._get("/field/bug", paramdict)
     def bug_get(self, bug_ids, aliases, paramdict):
+        bug_list = listify(bug_ids)
+        alias_list = listify(aliases)
         data = paramdict.copy()
-        data["id"] = listify(bug_ids)
-        data["alias"] = listify(aliases)
+
```

Comment:
Can you add a comment here explaining why we are doing this: basically it's a hack to keep existing getbug() and getbugs() semantics working in the higher level API

Reply:
Done :slightly_smiling_face: